apps/web/utils/outlook/label.ts

```diff
@@ -293,18 +319,21 @@ export async function archiveThread({
         logger.warn("Thread not found", { threadId, userEmail: ownerEmail });
         return { status: 404, message: "Thread not found" };
       }
-      logger.error(`Failed to move thread to ${folderName}`, {
+      logger.error(`Failed to move thread to folder ${folderId}`, {
         threadId,
         error,
       });
       throw error;
     }
 
     if (publishResult.status === "rejected") {
-      logger.error(`Failed to publish action to move thread to ${folderName}`, {
-        threadId,
-        error: publishResult.reason,
-      });
+      logger.error(
+        `Failed to publish action to move thread to folder ${folderId}`,
```

Comment:
i've made this note before. if you're adding logs, then don't use variables in it. harder to analyse logs this way